src/DataImport/gb_parse.py

```diff
@@ -26,6 +27,8 @@ def parse_genbank(genbank_file):
                 # uses filename (without extension) as species name
                 species = os.path.splitext(os.path.basename(genbank_file)[0])
 
+            print species
```

Comment:
Mostly so that I can see it working - this function takes a long time to run. No larger purpose.